indexer/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:20-alpine AS node
 FROM node AS api
 WORKDIR /assets
 COPY api/gear/vft_manager.idl .
-COPY api/gear/erc20_relay.idl .
+COPY api/gear/ethereum_event_client.idl .

```

Comment:
I suppose it isn't required in indexer anymore? `Relayed` event from `historical-proxy` should be enough

Reply:
Ah, I see it was changed in #237, so I think you can revert changes to indexer in this PR and wait for #237 to be merged